<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color:rgb(68, 68, 68);
	color:rgb(255, 255, 255);
	font: 12pt;
}
QMenuBar {
	background-color:rgb(200, 200, 200);
	color:rgb(60, 60, 60);
}
QWidget:checked {
	background-color:rgb(150, 150, 150);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="OpenGLWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>551</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>768</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionShow"/>
   <addaction name="act_DrawCube"/>
   <addaction name="act_DrawBall"/>
   <addaction name="act_DrawCone"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionMove"/>
   <addaction name="act_MoveCube"/>
   <addaction name="act_MoveBall"/>
   <addaction name="act_MoveCone"/>
  </widget>
  <action name="actionShow">
   <property name="text">
    <string>Draw:</string>
   </property>
  </action>
  <action name="act_DrawCube">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cube</string>
   </property>
  </action>
  <action name="act_DrawBall">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ball</string>
   </property>
  </action>
  <action name="act_DrawCone">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cone</string>
   </property>
  </action>
  <action name="actionMove">
   <property name="text">
    <string>Move:</string>
   </property>
  </action>
  <action name="act_MoveCube">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cube</string>
   </property>
  </action>
  <action name="act_MoveBall">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ball</string>
   </property>
  </action>
  <action name="act_MoveCone">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cone</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">openglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
